lib/rollbar/rake_tasks.rb

```diff
@@ -1,154 +1,7 @@
-require 'rollbar'
-begin
-  require 'rack/mock'
-rescue LoadError
-  puts 'Cannot load rack/mock'
-end
-require 'logger'
-
 namespace :rollbar do
   desc 'Verify your gem installation by sending a test exception to Rollbar'
   task :test => [:environment] do
+    require './verify_setup'
```

Comment:
This didn't work for me when executing `rake rollbar:test` from a Rails app. It takes the path relative to the app rather than rollbar-gem, and the require fails.